### ISSUE TITLE
bug: `include_size_comparison` not filtering correctly

### ISSUE THREAD

**Issue by Eirmas** [exoego/esbuild-bundle-analyzer#87]
`include_size_comparison` doesnt seem to be filtering like its supposed to.

I am using this config:

```yaml
      - name: Analyze esbuild bundle size
        uses: exoego/esbuild-bundle-analyzer@v1
        with:
          metafiles: 'dist/apps/blend/stats.json'
          show_details: false
          include_size_comparison: 'total'
          top_n_largest_paths: 0
          show_no_change: false
```

With this config, we are not getting a `total` row, and it only shows `Added` / `Deleted` rows. It does seem to filter away `No changes` correctly.

**Comment by exoego** [comment]
Thanks.
I'll take a look within a few days

**Comment by exoego** [comment]
This is fixed in v1.5.2:

<img width="472" alt="image" src="https://github.com/user-attachments/assets/9110fdb8-b62f-4e35-b28e-2ee4a179dd48">
